force_gromacs/commands/gromacs_commands.py

```diff
@@ -121,3 +121,47 @@ def __init__(self, name=None, flags=None, mpi_run=False,
         super(Gromacs_mdrun, self).__init__(
             name=name, *args, **kwargs
         )
+
+
+class Gromacs_select(BaseGromacsCommand):
```

Comment:
This command is called as `g_select` in versions of Gromacs < 5.0, however, it is changed to `gmx select` in later versions.